osrf_testing_tools_cpp/cmake/osrf_testing_tools_cpp_get_googletest_versions.cmake

```diff
@@ -31,17 +31,17 @@ macro(osrf_testing_tools_cpp_get_googletest_versions output_versions output_loca
   # Manually maintained list of googletest versions provided by this package.
   set(${output_versions}
     1.7.0
-    1.8.0
+    1.8.1
   )
   # Manually maintained list of googletest locations in this package, indexed the same as versions.
   # They are relative to the VENDOR_DIR, which is configurable.
   set(${output_locations}
     "google/googletest/release-1.7.0.tar.gz"
-    "google/googletest/release-1.8.0.tar.gz"
-  )
+    "google/googletest/release-1.8.1.tar.gz"
+    )
```

Comment:
nitpick: This whitespace change looks unnecessary and incorrect I think. Can you please undo it?